tests/run_tests.py

```diff
@@ -42,15 +44,15 @@
     "multiple_ssbo_update_with_graphics_pipeline.vkscript",
     "multiple_ubo_update_with_graphics_pipeline.vkscript",
     # DXC not currently building on bot
-    "draw_triangle_list_hlsl.amber",
+    DXC_TEST_CASE,
```

Comment:
Remove these from this set of SUPPRESSIONS and make a SUPPRESSIONS_DXC similar to SUPPRESSIONS_DAWN

Reply:
I followed the behavior of `--use-opencl`. PTAL.